contracts/Wrapped1155Factory.sol

```diff
@@ -135,16 +145,32 @@ contract Wrapped1155Factory is ERC1155Receiver {
             
             // assign name
             hex"7f",
-            "Wrapped ERC-1155", uint128(32),
+            // Chars in hex or UTF-8 = 8 bits => 2 hex numbers => 
+            // string + uint128(30) for 16 chars needs to padding zeros 
+            // and the length of the string
+            // leading zeros = (64 - (tokenName.length * 2)) / 2
+            // Wrapped ERC-1155GnosisProtocol = 31 * 2 zeros + length * 2 in hex
+            // (62 - (31 * 2)) = 0 zeros + length * 2 in hex
+            // "Wrapped ERC-1155", uint128(32), 
+            // Wrapped ERC-1155 = 16 zeros + length * 2 in hex
+            // WrappedERC-1155 = 15 * 2 zeros + length * 2 in hex
+            // (62 - (15 * 2)) = 16
+            //"WrappedERC-1155", hex"000000000000000000000000000000001E", 
+            tokenName, tokenNameOffset, 
             hex"600655",
             
             // assign symbol
             hex"7f",
+            // Chars in hex or UTF-8 = 8 bits => 2 hex numbers => 
+            // string + uint128(30) for 16 chars needs to padding zeros 
+            // and the length of the string
             "WMT", uint232(6),
             hex"600755",
             
             // assign decimals
             hex"60",
+            // push one byte
+            // min 1, max 2^8
```

Comment:
min is theoretically 0

Reply:
Well, yes, but 0x60 is a push instruction for at least 1 byte. Anyway, I will add the documentation.